resources/assets/less/beatmaps.less

```diff
@@ -264,6 +263,7 @@
 			position: absolute;
 			top: 45px;
 			left: 10px;
+			z-index: @z-index--beatmaps-panel - 2;
```

Comment:
wat ಠ_ಠ

This looks like defeating purpose of using centralized z-index definitions.